### PR TITLE
Explicitly check for cmake

### PR DESCRIPTION
**Pull request by malmaud**

### CODE REVIEW

**Comment by yuyichao** [comment]
Does this mean that the build process will raise an error if cmake is not found but the library is installed system-wise?

**Comment by malmaud** [comment]
Mmm ya, I guess you're right about that.

**Comment by tkelman** [comment]
add a shell script to do the checking with a nice error and have it run as the first build step?

**Comment by malmaud** [comment]
@tkelman how's this?

File: deps/cmake_check.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+command -v cmake > /dev/null
+
+if [[ $? -ne 0 ]]; then
```
Comment (yuyichao) [inline deps/cmake_check.sh]: FYI, this can be simplified to `if command -v cmake > /dev/null; then`.

**Comment by yuyichao** [comment]
LGTM.

**Comment by malmaud** [comment]
Thanks @yuyichao

**Comment by tkelman** [comment]
this shouldn't assume bash.

**Pull request merged**